### PR TITLE
Fix: some typos in documentation

### PR DESCRIPTION
**Pull request by helio-frota**

### CODE REVIEW

File: doc/addon.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The `Napi::Addon<T>` class can be used together with the `NODE_API_ADDON()` and
 class ExampleAddon : public Napi::Addon<ExampleAddon> {
  public:
   ExampleAddon(Napi::Env env, Napi::Object exports) {
-    // In the constructor we declare the functions the add-on makes avaialable
+    // In the constructor we declare the functions the add-on makes available
```
Comment (NickNaso) [inline doc/addon.md:37]: This fix is already in other PR. Could you please remove it?

**Review by NickNaso — COMMENTED** [review]
Just one comment. Thanks for the contribution.

File: doc/addon.md (hunk 1 above)
Comment (helio-frota) [reply doc/addon.md:37]: Sure thing :+1:

Comment (helio-frota) [reply doc/addon.md]: @NickNaso fix removed :+1:

**Review by NickNaso — APPROVED** [review]
Good work 👍

**Review by tniessen — APPROVED** [review]

**Review by jschlight — APPROVED** [review]
LGTM

**Review by mhdawson — APPROVED** [review]
LGTM

**Pull request merged**